scripts/php-xhprof.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Require repo deps
+/vagrant/scripts/epel.sh
+
+# install php-xhprof and graphviz
+yum install -y php-xhprof graphviz --enablerepo=epel --enablerepo=webtatic
+
+# clone the SS xhprof repo as we need that
+if [ ! -d /vagrant/xhprof ]; then
+	git clone https://github.com/sminnee/xhprof /vagrant/xhprof
```

Comment:
The _ss_environment include in Sam's code however, seems to fail for this setup.